ground/gcs/src/crashreporterapp/crashreporterapp.pro

```diff
@@ -3,10 +3,14 @@ QT       += core gui
 
 greaterThan(QT_MAJOR_VERSION, 4): QT += widgets
 
+CONFIG-=app_bundle
```

Comment:
would you prefer to move that below to the macx bracket to make sure it has no unintended side effects?